sda-commons-server-kafka-example/src/main/java/org/sdase/commons/server/kafka/KafkaExampleConsumerApplication.java

```diff
@@ -58,7 +57,7 @@ private void createExampleMessageListener(ObjectMapper configuredObjectMapper) {
     kafka.createMessageListener(
         MessageListenerRegistration.builder()
             .withDefaultListenerConfig()
-            .forTopicConfigs(Collections.singletonList(kafka.getTopicConfiguration("example0")))
+            .forTopic("exampleTopic")
```

Comment:
Do we also need to change the [`config.yaml`](https://github.com/SDA-SE/sda-dropwizard-commons/blob/fe3f06fd5a09c49b1634e625e9f7537e92773b1d/sda-commons-server-kafka-example/src/test/resources/test-config-consumer.yml)? Can we now remove `kafka.topics` completely?

Can you double-check that the `config` that was defined in the `config.yaml` was not used / is not important?